lib/razorpay.d.ts

```diff
@@ -1 +1,48 @@
-declare module "razorpay";
+import API, { IRazorpayHeaders } from './types/api'
+import customers from "./types/customers"
```

Comment:
Remove the imports if we are not adding the type.d.ts files for each module. 

And instead add `any`: 

```ts
  customers: any;
```